projects/storefrontlib/shared/components/item-counter/item-counter.component.html

```diff
@@ -7,7 +7,9 @@
 >
   -
 </button>
+<!-- TODO: (CXSPA-6034) Remove feature flag next major release -->
 <input
+  *cxFeatureLevel="'!6.8'"
```

Comment:
I don't think we need to feature flag this to remove the #qty binding since it is angular-specific and functionality is handled in the class. But lets still leave a comment to remove the binding later.

Reply:
Removed feature flag and left a comment